src/openms/include/OpenMS/KERNEL/Feature.h

```diff
@@ -183,6 +183,9 @@ namespace OpenMS
       return assignments;
     }
 
+    /// update ID referenes (primary ID, input matches) for this feature and any subfeatures
```

Comment:
can you add more documentation here for what this is used
e.g. use cases or example code.
like that I find it hard to make sense of that function here (without extensive researching into the ID datastructure)